apps/tunnel/src/main.c

```diff
@@ -72,6 +72,14 @@ static bool useSelectBased = false;
 
 static char* default_hosts[] = { "127.0.0.1", "localhost", 0 };
 
+uint8_t obscurity_key_id[] = {
```

Comment:
what is this?

Reply:
added as per customer request (for psk bootstrapping in a trusted setting (WPS-style)): They would like to obscure using a static key instead of sending psk as actual cleartext